app/views/landing-pages/parents/PageParents.vue

```diff
@@ -98,9 +98,9 @@
       <div class="container">
         <div class="row">
           <div class="col-lg-12">
-            <h2 class="text-center pixelated invest-heading">
+            <h1 class="text-center pixelated invest-heading">
```

Comment:
Selector is no longer in use and can be removed.

```suggestion
            <h1 class="text-center pixelated">
```